manifests/prometheus-rules.yaml

```diff
@@ -1782,6 +1802,18 @@ spec:
       for: 15m
       labels:
         severity: warning
+    - alert: PrometheusTargetLimitHit
```

Comment:
Hmm since this release of Prometheus that includes this change and this metric is not out yet, maybe we should pin Prometheus mixin to a release as well?

Reply:
sounds good, let me try to pin it to 2.20.

Reply:
@lilic pinned prometheus-mixin to `release-2.20` ptal, and thank you for the sharp eyes! :+1: